### ISSUE TITLE
Windows specific- Session Desktop quality of life requests

### ISSUE THREAD

**Issue by OlderNews** [oxen-io/session-desktop#2362]
Recommendation for optimal user experience on windows:
- When installing Session desktop, Offer app to pin to task bar, or start menu on install. This is a pretty standard Windows setting, and every time you install Session, it will remove the old task bar icon. So it would be ideal if when installing it replaced it, or offered it after each update. As it is deleted after each update. 

- Keep on in system tray should be default on, not often do people use a messaging application, and then close it every time they stop using it actively, as you will generally be waiting for a reply, or a call. It seems like the default should be on, rather than off for new users to have an optimal experience. 

- This was mentioned in a different feature request, but start on boot, would be great, this is something that usually comes up on install. (I put it here also, to bring all the windows improvements requests together) 

Thanks team :)

**Comment by KeeJef** [comment]
Yeah i'd like to do this as part of some upcoming on-boarding changes, this would probably help boost app usage aswell

**Comment by Bilb** [comment]
About the first one
> it will remove the old task bar icon 

this looks to be a not so resolve issue with electron-builder https://github.com/electron-userland/electron-builder/issues/2514
Looks like some people tried some stuff but it created other bugs. It would be nice to give for instance a build done with 
`allowToChangeInstallationDirectory: false` a try to see how that goes.
Might not be an easy fix looking at how many people still have it on that issue link above.

> Keep on in system tray should be default on
Easy to do. Keeping that issue as good first issue for that case


> auto start on boot
looks like electron does it badly and another package does it reliably: https://stackoverflow.com/questions/46318177/how-to-use-auto-launch-to-start-app-on-system-startup

**Comment by keybreak** [comment]
> Keep on in system tray should be default on

:100: that's sane defaults!

**Comment by KeeJef** [comment]
Tracking request 1st and 3rd request via SES-2302

**Comment by KeeJef** [comment]
Tracking 2nd request via SES-2303
